lib/publiccloud/utils.pm

```diff
@@ -859,4 +860,79 @@ sub detect_worker_ip {
     die "Worker IP could not be determined - return was $ip";
 }
 
+
+
+=head2 trup_call_remote 
+
+    trup_call_remote([timeout, exit_code, proceed_on_failure])
+
+    Run a transactional command on remote machine via ssh, non-interactive
+
+    Return: 
+    ($out, $res): a string with the full screen output and the number of final exit code: 
+
+=cut
+
+sub trup_call_remote {
+    my ($self, $cmd, %args) = @_;
+    my $start = time();
+    my $script = "sudo transactional-update -n $cmd";
+    my $timeout = $args{timeout} // 600;
+    my $exit = $args{exit_code} // 0;
+    my $proceed = $args{proceed_on_failure} // 0;
+    my ($out, $res);
+    # Check output just before the reboot start
+    if ($cmd =~ /reboot / && $args{exit_code} == 0) {
```

Comment:
We can drop the whole `if` block if we're not calling `trup_call_remote("reboot")`